docs/developers/setup.md

```diff
@@ -21,12 +21,12 @@ sbt tests/test
 ```
 
 The `--repo=<repository name>` option should match the name of your GitHub
-repository with dashes replaced by spaces.
+repository with dashes replaced by spaces. For
+`github.com/organization/scalafix-enterprise-rules`, type
```

Comment:
https://github.com/scalacenter/scalafix.g8/pull/19#discussion_r634245310

Reply:
I completely agree on this new naming strategy